bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/update/thing-updates.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="unifi:accessPoint">
+		<instruction-set targetVersion="1">
+			<add-channel id="online">
+				<type>unifi:online</type>
+				<description>Online status of the device</description>
+			</add-channel>
+			<add-channel id="state">
+				<type>unifi:devState</type>
+			</add-channel>
+			<add-channel id="name">
+				<type>unifi:name</type>
+				<description>Name of the device</description>
+			</add-channel>
+			<add-channel id="site">
+				<type>unifi:site</type>
+			</add-channel>
+			<add-channel id="ipAdress">
+				<type>unifi:ipAdress</type>
```

Comment:
Last finding, I hope:
```suggestion
			<add-channel id="ipAddress">
				<type>unifi:ipAddress</type>
```

I'm wondering what happened when you tested this? It should have failed.

Reply:
True... it failed. All other channels were updated though so I did not realize this one was missing 😱

Reply:
Typical German mistake...

🇩🇪 Adresse
🇺🇸 Address

Reply:
Also "adresse" in Danish, we have the same similarity struggle. 😀